### PR TITLE
Development

### PR DESCRIPTION
**Pull request by JoseGrangerSalgado**
Finished client-side of my portafolio webpage

### CODE REVIEW

**Review by josueetcom — COMMENTED** [review]
Hey, it looks like you have an uncompressed version of the jQuery library, which is making the diff rather large. This can result in big diffs later if you ever want to update the library. I suggest [including it from a CDN](https://www.w3schools.com/jquery/jquery_get_started.asp) instead. This'll also keep your repo smaller and limit the changes in this PR.

**Comment by JoseGrangerSalgado** [comment]
I understand,  I'll change it so that it downloads through the CDN

**Comment by JoseGrangerSalgado** [comment]
Done

File: portfolio/src/main/webapp/css/mainstyles.css [hunk 1]
```diff
@@ -638,6 +570,80 @@ hr{
 }
 }
 
+.loader-wrapper{
```
Comment (josueetcom) [inline portfolio/src/main/webapp/css/mainstyles.css:573]: Was this file sorted? Just curious.

**Review by josueetcom — APPROVED** [review]
Looks good to me 👍

File: portfolio/src/main/webapp/css/mainstyles.css (hunk 1 above)
Comment (JoseGrangerSalgado) [reply portfolio/src/main/webapp/css/mainstyles.css:573]: I don't think so no, I am still researching how to do it correctly.

**Pull request merged**